package.json

```diff
@@ -47,6 +47,10 @@
     "create-react-class": "15.5.3",
     "creditcards": "2.1.2",
     "cross-env": "4.0.0",
+    "d3-array": "1.2.0",
```

Comment:
hmm, perhaps the previous generator wasn't on 3.10.10 then. odd. nothing big to worry about though.